src/cdk/drag-drop/directives/drag.spec.ts

```diff
@@ -798,22 +847,17 @@ describe('CdkDrag', () => {
       let currentTime = 0;
 
       const fixture = createComponent(StandaloneDraggable);
-      fixture.componentInstance.dragStartDelay = '1000';
+      fixture.componentInstance.dragStartDelay = '500';
```

Comment:
The test used to:
1. Click (T 0)
2. Move the mouse (T 750)
3. Expect that the element did not move (T 750)
4. Move the mouse again (T 1250)
5. Expect the object to have moved (T 1250)

The test does not reflect the new behaviour I am suggesting. The first mouse move, because it is performed before the drag start delay, now cancels the drag sequence. The moving before the delay is elapsed is tested [here](https://github.com/angular/components/pull/16228/files#diff-fcb7ef452d47367765744a0eb1e4676bR775). This test ensures that moving after the delay is elapsed moves the element.